src/components/Home.jsx

```
@@ -0,0 +1,173 @@
+//           </div>
+//         </section>
+//   )
+// }
+
+
+
+import React from 'react';
+import { FaEnvelope, FaDownload } from 'react-icons/fa';
+import { Helmet } from 'react-helmet-async';
+
+export default function Home({ language, scrollToSection }) {
```

ESLint

[error] 'scrollToSection' is defined but never used.

(no-unused-vars)